newIDE/app/src/AssetStore/AssetCard.js

```diff
@@ -51,6 +56,14 @@ type Props = {|
   onOpenDetails: () => void,
 |};
 
+const isPixelArt = assetShortHeader => {
```

Comment:
As this reused let's move this to GDevelopServices/Asset.is

In the future we could change this so that pixel art asset are marked by the server :)